skins/laika/src/store/util.ts

```diff
@@ -1,5 +1,6 @@
 import { Validity } from '@/view_models/Validity';
-import { PostData, FormData } from '@/view_models/Address';
+import { FormData, PostData } from '@/view_models/Address';
```

Comment:
? :laughing:

Reply:
I blame PHPStorm for autistically re-organizing that :man_shrugging: